src/dfs/ipfs.ts

```diff
@@ -184,7 +184,7 @@ export class Ipfs extends Logger implements DfsInterface {
    *
    * @return     data as text
```

Comment:
```suggestion
   * @return     data as text or raw buffer
```

Also maybe
```
   * @param      returnBuffer  should the function return the raw buffer (default false)
```
for the argument documentation?